gtfs_realtime_translators/translators/njt_rail.py

```diff
@@ -74,7 +75,8 @@ def __make_trip_updates(cls, data):
                                                 headsign=headsign,
                                                 track=track,
                                                 block_id=block_id,
-                                                agency_timezone='America/New_York')
+                                                agency_timezone='America/New_York',
```

Comment:
Small non-blocker, but I think a nice pattern is to set the timezone for a translator as a constant class attribute where applicable